hc/api/tests/test_send_reports.py

```diff
@@ -0,0 +1,57 @@
+from datetime import timedelta
+
+from django.utils import timezone
+from hc.api.management.commands.sendreports import Command
+from hc.api.models import Check, Channel
+from hc.test import BaseTestCase
+
+
+class SendReportsTestCase(BaseTestCase):
+
+    def test_handle_one_run_sends_daily_report(self):
+
+        # Daily report for alice
+        # Set alice's join date to 1 day before
+        one_day_before = timezone.now() - timedelta(days=1)
+        self.alice.date_joined = one_day_before
+        self.profile.report_duration = 1
```

Comment:
What is this statement meant to do? I'm asking because `Profile` doesn't have a "report_duration" attribute.